frontend/package.json

```diff
@@ -51,6 +51,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "proxy": "http://localhost:3000"
```

Comment:
I don't think this was necessary because we explicitly give Apollo the full URL of the endpoint. This would be helpful if we called our API from the same host, like `http://localhost:8080/api/do/something`.